src/javascript/app/pages/trade/digit_ticker.js

```diff
@@ -0,0 +1,101 @@
+const DIGIT_BLOCK_SIZE        = 36;
+const DIGIT_BLOCK_LEFT_OFFSET = 66;
+const DIGIT_BLOCK_CORRECTION  = 2;
```

Comment:
I don't think we do this, all constants for this should be inside DigitTicker, same as the digits array, and should be in lowercase